src/scm/git.go

```diff
@@ -161,3 +163,16 @@ func (g *git) Checkout(revision string) error {
 	}
 	return nil
 }
+
+func (g *git) CurrentRevDate(format string) string {
+	out, err := exec.Command("git", "show", "-s", "--format=%ct").CombinedOutput()
+	if err != nil {
+		return "UNKNOWN"
+	}
+	timestamp, err := strconv.ParseInt(string(out), 10, 64)
+	if err != nil {
+		return string(out)
```

Comment:
I think I see the logic, but might be worth adding a comment about why it's OK to "ignore" the error here.

Reply:
I've actually made this return the error